### DISCUSSION TITLE
GitHub Copilot chat can't input Chinese with Microsoft Pinyin  in VS 2022

### DISCUSSION THREAD

**Discussion by Gsonovb** [community/community#65526] [Copilot Conversations]
### Select Topic Area

Bug

### Body

HI

I'm in VS2022(17.7.2) Can't enter Chinese in Chat window,

The IME prompt box will not appear, and the input content will only be in English.

Now the Chat plugin version is v0.1.1305.23247

BUT IN VSCODE I CAN ENTER Chinese.

**Comment by Akash1134** [comment]
Hi @Gsonovb ,

Thanks for reporting the issue. We have shared it internally for a review. We might reach out to you for more details about the issue you've reported.

**Reply by Marsirrr** [reply]
Hi, @Akash1134 
I also have this problem. When I first installed Copilot Chat, the input box can input Chinese, but I don’t know when it suddenly can’t

**Reply by shaozeye** [reply]
I also have the same problem, after uninstalling and reinstalling, I still can’t input Chinese

**Reply by Dandmoth** [reply]
Check the language preferences it's probably set to system

**Reply by Gsonovb** [reply]
I've tried using an older version of Microsoft Pinyin IME, but I still can't enter Chinese

**Reply by turqTanza** [reply]
I think this issue is similar following issues:
https://developercommunity.visualstudio.com/t/GitHub-Copilot-Chat:-IME-problem-in-typi/10456415
https://developercommunity.visualstudio.com/t/When-I-enter-Japanese-question-text-inli/10448400

The status of these issues is "Pending Release".

**Reply by turqTanza** [reply]
Yesterday this bug in Japanese IME was fixed. (0.1.1668.35657)
I think it was fixed in the Chinese IME as well, can you confirm it?

**Reply by Marsirrr** [reply]
(0.1.1668.35657) fixed this bug!

**Reply by Gsonovb** [reply]
Yes, it's fixed ! Thanks
